IAM/Frequently_asked_questions.mediawiki

```diff
@@ -211,11 +211,58 @@ If you own a volunteer LDAP account, then you can upgrade your mozillians profil
 
 ==== '''Q''': ''What issues I might encounter by upgrading to Firefox Accounts?'' ====
 
-There are some known issues with using Firefox Accounts in Mozilla IAM: 
+There are some known issues with using Firefox Accounts in Mozilla IAM:
 
 1.  Login with Firefox Accounts is unavailable from inside some Android applications, including IRCCloud and Slack. This is due to lack of localStorage support in some Android WebViews.
 2.  Firefox Accounts can only be used in Mozilla IAM with 2FA enabled. Note that once you choose to use Firefox Accounts, it is required to set up 2FA, to avoid being locked out of your account.
 
+==== '''Q''': ''Why do the email login links expire after 15 minutes?'' ====
+
+When you login using an email link, that link is valid for 15 minutes from when you request it.
+This expiration window of 15 minutes is driven both by security considerations and a desire for
+a positive user experience. The link is short lived so that there is a limited window
+of time during which a potential attacker could use the link if they were able to get access
+to it. This is especially important due to the inherently insecure nature of email
+transmission. 
+
+During the past 18+ months, experience shows that for a vast majority of users, this 15 minute expiration window has no effect on
+them as they receive the email link in their inbox mere seconds after they click the
+button requesting the link. Some users however do not receive the email login link
+immediately.
+
+Short delays in delivery are just part of how email delivery works.
+Longer delays however can be caused by a feature called [https://en.wikipedia.org/wiki/Greylisting Greylisting]. Continue on to the question below for further information on Greylisting.
+
+==== '''Q''': ''What is email Greylisting?'' ====
+Some email providers institute Greylisting on all inbound email for their users
+as a measure to reduce spam. Greylisting temporarily rejects email from mail servers
+where the sender hasn't communicated with the recipient before or due to some other
+signal indicating the email may be spam. A mail server that is Greylisting expects
+that a valid sender and mail server will continue to retry sending the email over
+time and the Greylisting mail server will eventually accept the mail. In the case
+of a "transactional" email like the email login link,
+[https://en.wikipedia.org/wiki/Greylisting#Delayed_delivery_issues Greylisting prevents users from being able to do a real-time login].
+
+ ==== '''Q''': ''What can I do about my email provider's Greylisting?'' ==== 
+Users who have email providers that use Greylisting will likely see this type of
```

Comment:
On line 262, it suggests to a user to have Mozilla whitelisted. I would suggest that this be expanded upon with specific information about about domains need to be whitelisted, or a link to a separate FAQ item along the lines of "how do I get Mozilla whitelisted with my e-mail provider", which lists specific domains that need to be whitelisted.